src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -651,6 +658,78 @@ handle_call(exchange, _From, #state{store=Store}=State) ->
     Pid = spawn(fun() -> do(fold, [Store, Function, []]) end),
     {reply, {ok, Pid}, State};
 
+handle_call(delta_gc, _From, #state{store=Store}=State) ->
+    Function =
+        fun({Id, #dv{delta_map=DeltaMap0, delta_ack_map=AckMap0,
+                     delta_counter=Counter}=_Object},
+            Acc0) ->
+                case orddict:is_empty(DeltaMap0) of
+                    true ->
+                        Acc0;
+                    false ->
+                        %% Remove disconnected or slow node() entries from the AckMap.
```

Comment:
After they've been removed and they attempt to sync, are they sent the entire state?